cmd/osv-scanner/main.go

```diff
@@ -155,6 +159,17 @@ func run(args []string, stdout, stderr io.Writer) int {
 				}
 			}
 
+			if context.Bool("experimental-licenses-summary") && context.IsSet("experimental-licenses") {
+				return fmt.Errorf("both --experimental-licenses-summary and --experimental-licenses flags set")
+			}
+			allowlist := context.StringSlice("experimental-licenses")
+			if context.IsSet("experimental-licenses") &&
+				(len(allowlist) == 0 ||
+					(len(allowlist) == 1 && allowlist[0] == "")) {
+				return fmt.Errorf("--experimental-licenses flags is empty")
```

Comment:
I think this reads a bit better:

```suggestion
				return fmt.Errorf("--experimental-licenses requires at least one value")
```

(feel free to chuck in "non-empty" if you like, but I personally think it's reasonable for us to say that `''` is "empty")